testdata/online-repositories-list-tests.json

```diff
@@ -0,0 +1,113 @@
+{
+    "repositoryTests" :
+    [
+        {
+            "owner": "git-fixtures",
+            "repository": "basic",
+            "version": 0,
+            "url": "https://github.com/git-fixtures/basic",
+            "createdAt": "2015-03-31 11:42:21 +0000 UTC",
+            "isPrivate": false,
+            "isArchived": false,
+            "hasWiki": false,
+            "numOfPrs": 2,
+            "numOfPrComments": 4
```

Comment:
Since we are adding a new file, either I would remove these entries not used from all the repository entries.